libs/loggers/csv_logger.py

```diff
@@ -55,68 +21,23 @@ def __init__(self, config, camera_id):
-        object (person) ids, bounding box coordinates and frame number.
-
-        Args: objects_list: A list of dictionary where each dictionary stores information of an object (person) in a
-        frame. frame_number: current frame number file_path: log file path
-        """
-        if len(objects_list) != 0:
-            object_dict = list(map(lambda x: prepare_object(x, frame_number), objects_list))
-
-            if not os.path.exists(file_path):
-                with open(file_path, "w", newline="") as csvfile:
-                    field_names = list(object_dict[0].keys())
-                    writer = csv.DictWriter(csvfile, fieldnames=field_names)
-                    writer.writeheader()
-
-            with open(file_path, "a", newline="") as csvfile:
-                field_names = list(object_dict[0].keys())
-                writer = csv.DictWriter(csvfile, fieldnames=field_names)
-                writer.writerows(object_dict)
-
-    def log_distances(self, distances, frame_number, file_path):
-        """Write violated incident's information of a frame into the object log file.
-
-        Each row of the distances log file consist of a violation information such as object (person) ids,
-        distance between these two object and frame number.
-
-        Args:
-            distances: A 2-d numpy array that stores distance between each pair of objects.
-            frame_number: current frame number
-            file_path: The path for storing log files
-        """
-        violating_objects = extract_violating_objects(distances, self.dist_threshold)
-        if not os.path.exists(file_path):
-            with open(file_path, "w", newline="") as csvfile:
-                field_names = ["frame_number", "object_0", "object_1", "distance"]
-                writer = csv.DictWriter(csvfile, fieldnames=field_names)
+        file_path = os.path.join(self.objects_log_directory, file_name + ".csv")
+        file_exists = os.path.isfile(file_path)
+        with open(file_path, "a") as csvfile:
+            headers = ["Version", "Timestamp", "DetectedObjects", "ViolatingObjects",
+                       "EnvironmentScore", "Detections", 'ViolationsIndexes']
+            writer = csv.DictWriter(csvfile, fieldnames=headers)
+
+            if not file_exists:
                 writer.writeheader()
-        with open(file_path, "a", newline="") as csvfile:
-            field_names = ["frame_number", "object_0", "object_1", "distance"]
-            writer = csv.DictWriter(csvfile, fieldnames=field_names)
-            writer.writerows([{"frame_number": frame_number,
-                               "object_0": indices[0],
-                               "object_1": indices[1],
-                               "distance": distances[indices[0], indices[1]]} for indices in violating_objects])
+
+            writer.writerow(
+                {'Version': version, 'Timestamp': time_stamp, 'DetectedObjects': detected_objects_cout,
```

Comment:
You are using single quotes here. Do we want to use " everywhere?

Reply:
I changed the quotes from `'` to `"`. I also apply the change in the `web_hook_logger.py `